docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc

```diff
@@ -0,0 +1,22 @@
+== Terraform Module Scanning
+
+Prisma Cloud supports rendering and scanning both public, private, and locally cached Terraform modules (modules). This capability enables users to analyze misconfigurations that may result from the module definition and the variables defined in the module block.
+
+image::application-security/tf-module-scan.png[]
+
+=== Public and Locally Cached Modules
+
+Automated support for public modules is provided by the platform for all version control system (VCS) repositories that have been onboarded (For information on onboarding repositories refer to xref:../../get-started/connect-code-and-build-providers/code-repositories/code-repositories.adoc[Code Repositories]).
```

Comment:
Automated support for public modules is provided by the platform for all Version Control System (VCS) repositories that have been onboarded. For information on onboarding repositories refer to xref:../../get-started/connect-code-and-build-providers/code-repositories/code-repositories.adoc[Code Repositories].

Reply:
'version control systems': it is common to use lowercase letters, as it is a generic term. For example: "Git is a version control system"